examples/mms/asr/tutorial/MMS_ASR_Inference_Colab.ipynb

```diff
@@ -0,0 +1,332 @@
+            "  Metadata:\n",
+            "    encoder         : Lavf58.45.100\n",
+            "  Duration: 00:00:06.88, start: 0.069063, bitrate: 24 kb/s\n",
+            "    Stream #0:0: Audio: mp3, 16000 Hz, mono, fltp, 24 kb/s\n",
+            "Stream mapping:\n",
+            "  Stream #0:0 -> #0:0 (mp3 (mp3float) -> pcm_s16le (native))\n",
+            "Press [q] to stop, [?] for help\n",
+            "Output #0, wav, to './audio_samples/audio.wav':\n",
+            "  Metadata:\n",
+            "    ISFT            : Lavf58.29.100\n",
+            "    Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 16000 Hz, mono, s16, 256 kb/s\n",
+            "    Metadata:\n",
+            "      encoder         : Lavc58.54.100 pcm_s16le\n",
+            "size=     213kB time=00:00:06.80 bitrate= 256.1kbits/s speed= 254x    \n",
+            "video:0kB audio:212kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.035846%\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 4: Run Inference and transcribe your audio(s)\n"
+      ],
+      "metadata": {
+        "id": "44UvHjmMI28Z"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "\n",
+        "os.environ[\"TMPDIR\"] = '/content/temp_dir'\n",
+        "os.environ[\"PYTHONPATH\"] = \".\"\n",
+        "os.environ[\"PREFIX\"] = \"INFER\"\n",
+        "os.environ[\"HYDRA_FULL_ERROR\"] = \"1\"\n",
+        "os.environ[\"USER\"] = \"micro\"\n",
+        "\n",
+        "!python examples/mms/asr/infer/mms_infer.py --model \"/content/fairseq/models_new/mms1b_fl102.pt\" --lang \"eng\" --audio \"/content/fairseq/audio_samples/audio.wav\"\n"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "J8N1RKtBiw5V",
+        "outputId": "4280b307-d4e2-441f-dd92-e92ca30bf107"
+      },
+      "execution_count": 12,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            ">>> preparing tmp manifest dir ...\n",
+            ">>> loading model & running inference ...\n",
+            "2023-05-24 23:57:30.372610: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.\n",
+            "To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.\n",
+            "2023-05-24 23:57:32.231893: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT\n",
+            "===============\n",
+            "Input: /content/fairseq/audio_samples/audio.wav\n",
+            "Output: a tornado is a spinning colum of very low-pressure air which sucks it surrounding air inward and upward\n"
+          ]
+        }
+      ]
```

Comment:
Can you give an example for how to extend it to non English languages.

Reply:
Please also mention at the end that LM decoding can be done to further improve performance.